### PR TITLE
Update the state sync doc

### PR DESCRIPTION
**Pull request by kmurphypolygon**
Quick tidy of the state sync doc.

### CODE REVIEW

File: docs/zkEVM/get-started/state-sync-snapshots.md [hunk 1]
```diff
@@ -2,47 +2,72 @@
 comments: true
 ---
 
-Use snapshots for quick state syncing. 
+!!! important
+    New and streamlined state sync process coming online soon.
 
-Find the zkEVM mainnet snapshots at the below urls.
+Follow the instructions below to run a fast state sync on a database using the relevant snapshot, file name given in the [snapshots section](#snapshots).
```
Comment (hsutaiyu) [inline docs/zkEVM/get-started/state-sync-snapshots.md]:
```suggestion
Follow the instructions below to run a fast state sync on a node database using the relevant snapshot. Snapshot file URLs are available in the [snapshots section](#snapshots).
```

File: docs/zkEVM/get-started/state-sync-snapshots.md [hunk 2]
```diff
@@ -2,47 +2,72 @@
...
 
-    ```bash
-    https://zkevm-mai.s3.eu-west-1.amazonaws.com/mainnet-bridgedb.sql.gz (315MB)
-    https://zkevm-mai.s3.eu-west-1.amazonaws.com/mainnet-statedb.sql.gz (6GB)
-    https://zkevm-mai.s3.eu-west-1.amazonaws.com/mainnet-hashdb.sql.gz (160GB)
-    ```
+## Sync a database
 
-Find the testnet snapshots at the below urls.
+1. Retrieve the back up file:
```
Comment (hsutaiyu) [inline docs/zkEVM/get-started/state-sync-snapshots.md]:
```suggestion
1. Run the following command to download the snapshot with the same name as given in the URL.
```

File: docs/zkEVM/get-started/state-sync-snapshots.md [hunk 3]
```diff
@@ -2,47 +2,72 @@
...
 
-    ```bash
-    https://zkevm-pub.s3.eu-west-1.amazonaws.com/testnet-bridgedb.sql.gz (312MB)
-    https://zkevm-pub.s3.eu-west-1.amazonaws.com/testnet-statedb.sql.gz (1.9GB)
-    https://zkevm-pub.s3.eu-west-1.amazonaws.com/testnet-hashdb.sql.gz (49GB)
+    ```sh
+    curl <snapshot-file> -o statedb.sql.gz
```
Comment (hsutaiyu) [inline docs/zkEVM/get-started/state-sync-snapshots.md]:
```suggestion
    curl <snapshot-url> -o <db-name>.sql.gz
```

File: docs/zkEVM/get-started/state-sync-snapshots.md [hunk 4]
```diff
@@ -2,47 +2,72 @@
...
+6. Start everything again
 
-    # Start everything
+    ```sh
     docker compose up -d
+    ```
+
+## Snapshots
+
+The snapshot for the bridge, state, and hash databases are given below for mainnet and testnet.
```
Comment (hsutaiyu) [inline docs/zkEVM/get-started/state-sync-snapshots.md:55]:
```suggestion
The snapshots for the bridge, state, and hash databases for both mainnet and testnet are given below.
```

**Review by hsutaiyu — APPROVED** [review]
Approved with suggestions

File: docs/zkEVM/get-started/state-sync-snapshots.md [hunk 5]
```diff
@@ -2,47 +2,72 @@
 comments: true
 ---
 
-Use snapshots for quick state syncing. 
+!!! important
+    New and streamlined state sync process coming online soon.
 
-Find the zkEVM mainnet snapshots at the below urls.
+Follow the instructions below to run a fast state sync on a node database using the relevant snapshot. Snapshot file URLs are available in the [snapshots section](#snapshots).
```
Comment (kmurphypolygon) [inline docs/zkEVM/get-started/state-sync-snapshots.md]:
```suggestion
Follow the instructions below to run a fast state sync on a node database using the relevant snapshot. Snapshot URLs are available in the [snapshots section](#snapshots).
```

File: docs/zkEVM/get-started/state-sync-snapshots.md [hunk 6]
```diff
@@ -2,47 +2,72 @@
...
 
-    ```bash
-    https://zkevm-mai.s3.eu-west-1.amazonaws.com/mainnet-bridgedb.sql.gz (315MB)
-    https://zkevm-mai.s3.eu-west-1.amazonaws.com/mainnet-statedb.sql.gz (6GB)
-    https://zkevm-mai.s3.eu-west-1.amazonaws.com/mainnet-hashdb.sql.gz (160GB)
-    ```
+## Sync a database
 
-Find the testnet snapshots at the below urls.
+1. Run the following command to download the snapshot with the same name as given in the URL.
```
Comment (kmurphypolygon) [inline docs/zkEVM/get-started/state-sync-snapshots.md]:
```suggestion
1. Run the following command to download the snapshot with the correct `snapshot-url` and `db-name` as given in the URL.
```

File: docs/zkEVM/get-started/state-sync-snapshots.md [hunk 7]
```diff
@@ -2,47 +2,72 @@
...
-Find the testnet snapshots at the below urls.
+1. Run the following command to download the snapshot with the correct `snapshot-url` and `db-name` as given in the URL.
 
-    ```bash
-    https://zkevm-pub.s3.eu-west-1.amazonaws.com/testnet-bridgedb.sql.gz (312MB)
-    https://zkevm-pub.s3.eu-west-1.amazonaws.com/testnet-statedb.sql.gz (1.9GB)
-    https://zkevm-pub.s3.eu-west-1.amazonaws.com/testnet-hashdb.sql.gz (49GB)
+    ```sh
+    curl <snapshot-url> -o <db-name>.sql.gz
     ```
```
Comment (kmurphypolygon) [inline docs/zkEVM/get-started/state-sync-snapshots.md:16]:
```suggestion
    ```

   For example: 

   ```sh
   curl https://zkevm-pub.s3.eu-west-1.amazonaws.com/testnet-statedb.sql.gz -o statedb.sql.gz
   ```
```

File: docs/zkEVM/get-started/state-sync-snapshots.md [hunk 8]
```diff
@@ -2,47 +2,77 @@
...
 
-    ```bash
-    https://zkevm-mai.s3.eu-west-1.amazonaws.com/mainnet-bridgedb.sql.gz (315MB)
-    https://zkevm-mai.s3.eu-west-1.amazonaws.com/mainnet-statedb.sql.gz (6GB)
-    https://zkevm-mai.s3.eu-west-1.amazonaws.com/mainnet-hashdb.sql.gz (160GB)
-    ```
+## Sync a database
 
-Find the testnet snapshots at the below urls.
+1. Run the following command to download the snapshot with the correct `snapshot-url` and `db-name` as given in the URL.
```
Comment (kmurphypolygon) [inline docs/zkEVM/get-started/state-sync-snapshots.md]:
```suggestion
1. Run the following command to download the snapshot with the correct `snapshot-url` and `db-name`.
```

File: docs/zkEVM/get-started/state-sync-snapshots.md [hunk 9]
```diff
@@ -2,47 +2,77 @@
...
+3. Remove data from the database, keeping the schema, and a raw backup copy:
+
+    ```sh
     docker compose exec -it zkevm-state-db pg_dump -U state_user -v -Fc -s -f /tmp/statedb.schema state_db
     docker compose exec -it zkevm-state-db dropdb -U state_user state_db
     docker compose exec -it zkevm-state-db createdb -U state_user state_db
     docker compose exec -it zkevm-state-db pg_restore -U state_user -v -d state_db /tmp/statedb.schema
+    ```
+
+4. Import the backup file into database:
```
Comment (kmurphypolygon) [inline docs/zkEVM/get-started/state-sync-snapshots.md]:
```suggestion
4. Import the backup file into the database:
```

File: docs/zkEVM/get-started/state-sync-snapshots.md [hunk 10]
```diff
@@ -2,47 +2,77 @@
...
+
+### Testnet
+
+```bash
+https://zkevm-pub.s3.eu-west-1.amazonaws.com/testnet-bridgedb.sql.gz (312MB)
+https://zkevm-pub.s3.eu-west-1.amazonaws.com/testnet-statedb.sql.gz (1.9GB)
+https://zkevm-pub.s3.eu-west-1.amazonaws.com/testnet-hashdb.sql.gz (49GB)
 ```
+
+
```
Comment (kmurphypolygon) [inline docs/zkEVM/get-started/state-sync-snapshots.md]:
```suggestion
```

File: docs/zkEVM/get-started/state-sync-snapshots.md [hunk 11]
```diff
@@ -2,47 +2,76 @@
...
-    # Let only datatabase running
+2. Stop/start the database:
+    
+    ```sh
     docker compose stop
     docker compose start zkevm-state-db
+    ```
 
-    # Remove data from database, keeping schema (the backup is raw COPY)
+3. Remove data from the database, keeping the schema, and a raw backup copy:
```
Comment (kmurphypolygon) [inline docs/zkEVM/get-started/state-sync-snapshots.md]:
```suggestion
3. Remove data from the database, keeping the schema and a raw backup copy:
```

File: docs/zkEVM/get-started/state-sync-snapshots.md [hunk 12]
```diff
@@ -2,47 +2,76 @@
...
 
-    ```bash
-    https://zkevm-mai.s3.eu-west-1.amazonaws.com/mainnet-bridgedb.sql.gz (315MB)
-    https://zkevm-mai.s3.eu-west-1.amazonaws.com/mainnet-statedb.sql.gz (6GB)
-    https://zkevm-mai.s3.eu-west-1.amazonaws.com/mainnet-hashdb.sql.gz (160GB)
-    ```
+## Sync a database
 
-Find the testnet snapshots at the below urls.
+1. Run the following command to download the snapshot with the correct `snapshot-url` and `db-name`.
```
Comment (kmurphypolygon) [inline docs/zkEVM/get-started/state-sync-snapshots.md]:
```suggestion
1. Run the following command to download the snapshot with the correct `snapshot-url` and `db-name`:
```

File: docs/zkEVM/get-started/state-sync-snapshots.md [hunk 13]
```diff
@@ -2,47 +2,76 @@
...
+    ```
 
-    # Import the backup file into database
+4. Import the backup file into the database:
+
+    ```sh
     gunzip -c statedb.sql.gz | docker compose exec -T zkevm-state-db psql -U state_user -d state_db
+    ```
+
+5. Clean up
```
Comment (kmurphypolygon) [inline docs/zkEVM/get-started/state-sync-snapshots.md]:
```suggestion
5. Clean up:
```

File: docs/zkEVM/get-started/state-sync-snapshots.md [hunk 14]
```diff
@@ -2,47 +2,76 @@
...
+
+5. Clean up:
 
-    # Cleanup
+    ```sh
     docker compose exec zkevm-state-db rm /tmp/statedb.schema
     rm statedb.sql.gz
+    ```
+    
+6. Start everything again
```
Comment (kmurphypolygon) [inline docs/zkEVM/get-started/state-sync-snapshots.md]:
```suggestion
6. Start everything again:
```

**Pull request merged**